.github/workflows/plugins-ci-swagger.yml

```diff
@@ -0,0 +1,42 @@
+name: Plugin CI - Swagger
```

Comment:
I would not add a plugin specific workflow in here

Reply:
The plugin needs a step in which it prepares some additional files needed to run the test suite.

@mcollina > Should I remove it and change the npm scripts in `fastify-swagger` repo ?